### ISSUE TITLE
Stop show new tv-show from a week ago

### ISSUE THREAD

**Issue by Anian-igor** [croneter/PlexKodiConnect#544]
Hello.
I have PKC on my android TV, Windows and Android table. A week ago PKC stop show new tv-show. But film show correct. I trying to recreate database and totaly reinstall kodi on windows machine but with no luck. Maybe some new version of plex don't work properly.
I have PKC 2.4.3 and plex 1.13.8.5395

**Comment by croneter** [comment]
I need much more info than that. What do you mean with `PKC stop show new tv-show`?
* Is there NOTHING showing up in the Kodi TV show section?
* Do new episodes that you just downloaded and added to Plex not show up "instantly" in PKC?
* Please share your log: https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by Anian-igor** [comment]
I removed all NOTICE and DEBUG string before upload to pastebin
https://pastebin.com/CkCRtc55
But full log available via dropbox
https://www.dropbox.com/s/6fsm8bnu4pnf44k/kodi.log?dl=0

**Comment by croneter** [comment]
No that does not work - please ONLY remove the token itself, I need the log entries

**Comment by Anian-igor** [comment]
Log without token available via gist
https://gist.githubusercontent.com/Anian-igor/b84c5e4258bce1bab05670b94ffd33e0/raw/dca93caa7b97f1b7157086366963b8b2e2c2dac8/gistfile1.txt

**Comment by croneter** [comment]
Ough thanks, nice bug hunting!

**Comment by croneter** [comment]
What's the file path for your episode with plex id `11294` as well as `11293`? 

To do that: Use Plex Web, open ANY movie or episode to show its meta information, hit the three dots "..." at the top right, then "Get Info", then "Show XML". A new browser window opens. The URL of this new window will look like this:
```
https://192-168-0-2.<...>.plex.direct:32400/library/metadata/825032?checkFiles=1&includeExtras=1&includeBandwidths=1&X-Plex-Token=<token>
```
Simply replace the Plex id - `825032` in the example URL above - with `11294` and then `11293`. Please share both XMLs here like you shared your log file

**Comment by Anian-igor** [comment]
showid 11293
https://gist.github.com/Anian-igor/4edae5ef22c21976ca882c318446dd16
showid 11294 is absent, but I removed few tv-shows in last hour.

**Comment by croneter** [comment]
Please try the new beta version 2.4.6 and let me know

**Comment by Anian-igor** [comment]
Done. 
https://gist.githubusercontent.com/Anian-igor/9e321f1de8081776a436d53e7a1f1466/raw/0e0294fa08737cecbece28caaefad2c3fff8a65a/kodi.log

**Comment by croneter** [comment]
Which means what? Is it working now?

**Comment by Anian-igor** [comment]
Nope, I still have an issue.
I just got an log
https://gist.githubusercontent.com/Anian-igor/9e321f1de8081776a436d53e7a1f1466/raw/0e0294fa08737cecbece28caaefad2c3fff8a65a/kodi.log

**Comment by croneter** [comment]
Nothing in the log, sorry. Please do the following:
* Enable Kodi debug logging
* Reset the database: go to the PKC settings, Advanced, then "Reset the Kodi database" (in red). Hit YES, then No. 
* Let Kodi restart and resync. 
* Share your debug log file just after the sync completed

**Comment by croneter** [comment]
Even more important: could you please share the Plex XML file for your plex ids `38574` and `38688` as described above? ("What's the file path for your episode with plex id `11294` as well as `11293`?")

They seem really, really off 😢

**Comment by Anian-igor** [comment]
38574
https://gist.github.com/Anian-igor/dfa39e3ff299d8004b35ab775d22bd29
38688 
https://gist.github.com/Anian-igor/c0bcf2ba7ef5c19704abb8a42d964844

**Comment by Anian-igor** [comment]
Full kodi.log
https://gist.github.com/Anian-igor/9e321f1de8081776a436d53e7a1f1466

**Comment by Anian-igor** [comment]
Something strange. I can see a show on desk than start watching on other devices. But kodi didn't show it in a list before.

**Comment by croneter** [comment]
I don't understand. What do you mean with your last sentence?

**Comment by Anian-igor** [comment]
I found that I can see only 34 tv-show from total 55.
And tv-show that I cannot see in list on kodi shown "on desk" list. But If I started "continue watching" from kodi than progress didn't shown on plex server.

**Comment by croneter** [comment]
This is really weird - you're the first among literally thousands to have such issues... 

Could you please share the XML files (like above) for the plex IDs 11453 and 11452? They're season and TV show of the episodes above

**Comment by Anian-igor** [comment]
11453 
https://gist.github.com/Anian-igor/1172b90d3f4c8aa0b57c5926a4a74ad5
11452
https://gist.github.com/Anian-igor/7aea96961167b5ff3b3e7e50a0c7c590

**Comment by Anian-igor** [comment]
I created totally new Library with only one tv-show that contain one season and one tv-show. And pkc skip it.

**Comment by croneter** [comment]
Log file of that please - might help a lot

**Comment by Anian-igor** [comment]
Log file (I trying to use kodi 18 beta 3)
https://gist.github.com/Anian-igor/9e321f1de8081776a436d53e7a1f1466
Test library "tshow"
Item xml
https://gist.github.com/Anian-igor/6707ddff624ad2fd18962ce9dc08567e
Parent xml (season 1)
https://gist.github.com/Anian-igor/91538f8439e353455a7e428be9425e84
Grandparent xml (tv-show)
https://gist.github.com/Anian-igor/ea6f773c6fbfbbbdbc9e2824a8cc9029

**Comment by croneter** [comment]
The sync was not done yet when you took your log file (and quit Kodi?). Please repeat and make sure that the initial sync finishes

**Comment by croneter** [comment]
So for your setup right now, just reset the Kodi database in the PKC settings under Advanced, restart Kodi, let the sync finish. If the episode you shared the XML from does not show up, please share your log file again

**Comment by Anian-igor** [comment]
I hope this log is full. I tried to install kodi to same server there plex located.
https://gist.github.com/Anian-igor/9e321f1de8081776a436d53e7a1f1466

**Comment by croneter** [comment]
Quick update: I'm working on PKC **BETA** version 2.5.0, a GIANT update. It should hopefully solve your issues. No, I haven't published 2.5.0 yet 😉

**Comment by Anian-igor** [comment]
I moved to emby server already :(
